test-lib-openshift.sh

```diff
@@ -368,17 +377,97 @@ function ct_os_new_project() {
     return
   fi
   local project_name="${1:-$(_ct_os_get_uniq_project_name)}" ; shift || :
-  oc new-project "${project_name}"
-  # let openshift cluster to sync to avoid some race condition errors
-  sleep 3
-  if test -n "${OPENSHIFT_CLUSTER_PULLSECRET_PATH:-}" -a -e "${OPENSHIFT_CLUSTER_PULLSECRET_PATH:-}"; then
-    oc create -f "$OPENSHIFT_CLUSTER_PULLSECRET_PATH"
-    # add registry pullsecret to the serviceaccount if provided
-    secret_name=$(grep '^\s*name:' "$OPENSHIFT_CLUSTER_PULLSECRET_PATH" | awk '{ print $2 }')
-    oc secrets link --for=pull default "$secret_name"
+  if [ "${SHARED_CLUSTER:-false}" == 'true' ]; then
+    oc create -f - <<EOF
+apiVersion: tenant.paas.redhat.com/v1alpha1
```

Comment:
This can be stored as a file somewhere and we can replace only specific field. But where to store it?